qutebrowser/config/configdata.yml

```diff
@@ -1042,6 +1042,44 @@ editor.encoding:
   default: utf-8
   desc: Encoding to use for the editor.
 
+## fileselect
+
+fileselect.handler:
```

Comment:
Hmm, I wonder... should this be a global decision, or should we allow the user to select different handlers for the different kind of prompts (download / single upload / multiple upload)?

Reply:
If we don't plan on having any other methods than 'default' and 'external', we could have a `fileselect.external` list which contains types which should be called externally.

I agree this could be useful for people who have a file-picker than only handles single files